guides/introduction/Testing.md

```diff
@@ -0,0 +1,89 @@
+If you need to verify that your application correctly receives `ecto_watch` events, consider the following:  
+
+- **PostgreSQL, not Ecto, emits the notifications.** This means that for your test to capture notifications, the transaction that triggers them **must be committed**.
+
+- **By default, transactions in tests are not committed.** The [`Ecto.Adapters.SQL.Sandbox`](https://hexdocs.pm/ecto_sql/Ecto.Adapters.SQL.Sandbox.html) ensures test transactions are rolled back unless explicitly overridden.  
+
+- **To ensure notifications are sent, disable the sandbox mode for the test:** 
+```elixir
+Ecto.Adapters.SQL.Sandbox.checkout(MyRepo, sandbox: false)
+
+```
+This forces all database changes to be persisted, so notifications will be emitted. However, be mindful that you may need to manually clean up test records.
+
+### Example
+
+```elixir
+defmodule My.Application do
+  use Application
+
+  @impl true
+  def start(_type, _args) do
+    children = [
+     {EctoWatch,
+       repo: MyRepo,
+       pub_sub: MyPubSub,
+       watchers: [
+         {record, :updated,
+          trigger_columns: [:example_column], label: :notification}
+       ]}
+    ]
```

Comment:
```suggestion
defmodule MyApp.Application do
  use Application

  alias MyApp.Records

  @impl true
  def start(_type, _args) do
    children = [
     {EctoWatch,
       repo: MyRepo,
       pub_sub: MyPubSub,
       watchers: [
         {Records.Record, :updated}
       ]}
    ]
```

I took out `trigger_columns: [:example_column]` since it isn't used in the rest of the example.

I think we should take out the `label: :notification` as well and have the `handle_info` expect `{Records.Record, :updated}` because that's the base use case for `ecto_watch` that people start with.